src/OpenSpaceToolkit/Physics/Time/Time.cpp

```diff
@@ -154,6 +154,26 @@ Real Time::getFloatingSeconds() const
            (static_cast<double>(microsecond_) / 1e6) + (static_cast<double>(nanosecond_) / 1e9);
 }
 
+Real Time::getTotalFloatingSeconds() const
+{
+    if (!this->isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("Time");
+    }
+
+    return static_cast<double>(hour_) * 3600.0 + static_cast<double>(minute_) * 60.0 + this->getFloatingSeconds();
```

Comment:
Why the cast to a double? Doesn't this happen implicitly with the `Real` <> `double` conversion?

Reply:
hour_ is a Uint8 so you have to cast it to a double